main/cicd/cicd-pipeline/config/buildspec/buildspec.yml

```diff
@@ -21,6 +21,16 @@ phases:
       - echo "Downloading environment config"
       - ./scripts/download-env-config.sh "$DEPLOYMENT_BUCKET" "$ENV_NAME"
       - printf "\n\n"
+      - echo "Creating ~/.aws/credentials"
```

Comment:
I think you need to add something similar for `buildspec-int-test.yml`

Reply:
There are no `sls` commands in the script that build runs([run-integration-tests.sh](https://github.com/awslabs/service-workbench-on-aws/blob/mainline/scripts/run-integration-tests.sh)). Why do you think we need to do something similar?

Reply:
nevermind, you're right. Resolving this comment.